recipes/r-bigrquery/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = '0.4.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bigrquery
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: bigrquery_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/bigrquery_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/bigrquery/bigrquery_{{ version }}.tar.gz
+  sha256: 2168e9cdd76aa253f65ee3bf1670af7c4e19e52e3ef8c49c64a96241d12f111e
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-dbi
+    - r-assertthat
+    - r-httr
+    - r-jsonlite
+    - r-progress
+    - r-readr
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-dbi
+    - r-assertthat
+    - r-httr
+    - r-jsonlite
+    - r-progress
+    - r-readr
+    - {{native}}gcc-libs   # [win]
+    - libgcc               # [not win]
+
+test:
+  commands:
+    - $R -e "library('bigrquery')"  # [not win]
+    - "\"%R%\" -e \"library('bigrquery')\""  # [win]
+
+about:
+  home: https://github.com/rstats-db/bigrquery
+  license: GPL-3
+  summary: Easily talk to Google's 'BigQuery' database from R.
+  license_family: GPL3
```

Comment:
You will likely be asked to add a license file because this package is GPL'd. You can add the license file by adding the following two lines here:

```
  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'   # [unix]
  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3' # [win]
```

Reply:
Ok, lets add it!

Reply:
In the future this will be done automatically with the helper script: https://github.com/bgruening/conda_r_skeleton_helper/pull/8